ui/page/proposal/proposals_page.go

```diff
@@ -234,13 +234,11 @@ func (pg *ProposalsPage) initLayoutWidgets() {
 	pg.syncButton = new(widget.Clickable)
 
 	pg.infoIcon = pg.Icons.ActionInfo
-	pg.infoIcon.Color = pg.Theme.Color.Gray
 
 	pg.legendIcon = pg.Icons.ImageBrightness1
-	pg.legendIcon.Color = pg.Theme.Color.InactiveGray
+	// pg.legendIcon.Color = pg.Theme.Color.InactiveGray
```

Comment:
why is this commented?

Reply:
The update does not support icon color struct variable.. so it is no longer needed.